app/src/main/java/com/dsd/carcompanion/api/models/VehicleInfo.kt

```diff
@@ -0,0 +1,6 @@
+package com.dsd.carcompanion.api.models
+
+data class VehicleInfo(
+    val label: String,
```

Comment:
This label is never changable?

Reply:
No, it is just the description for each information, state can be changed but not label